Makefile.am

```diff
@@ -64,24 +64,16 @@ GZ_RULE = \
 
 SUBST_RULE = \
 	$(AM_V_GEN) $(MKDIR_P) $(dir $@) && sed \
+	-e 's,[@]PACKAGE[@],$(PACKAGE),g' \
+	-e 's,[@]admin_group[@],$(admin_group),g' \
 	-e 's,[@]datadir[@],$(datadir),g' \
+	-e 's,[@]group[@],$(COCKPIT_GROUP),g' \
 	-e 's,[@]libexecdir[@],$(libexecdir),g' \
-	-e 's,[@]sysconfdir[@],$(sysconfdir),g' \
-	-e 's,[@]libdir[@],$(libdir),g' \
-	-e 's,[@]includedir[@],$(includedir),g' \
-	-e 's,[@]bindir[@],$(bindir),g' \
-	-e 's,[@]sbindir[@],$(sbindir),g' \
-	-e 's,[@]prefix[@],$(prefix),g' \
-	-e 's,[@]exec_prefix[@],$(exec_prefix),g' \
 	-e 's,[@]prefix[@],$(prefix),g' \
-	-e 's,[@]PACKAGE[@],$(PACKAGE),g' \
-	-e 's,[@]VERSION[@],$(VERSION),g' \
```

Comment:
doc/guide/version.in uses `@VERSION`, and it IMHO makes sense there. Please keep this. I checked the others, and they are indeed unused.

Reply:
`doc/guide/version.in` is substituted from `configure.ac`, not the `$(SUBST_RULE)`.